tools/bumper-bot/parser/parser.go

```diff
@@ -0,0 +1,48 @@
+package parser
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"github.com/pkg/errors"
+
+	"gopkg.in/yaml.v2"
+)
+
+type Component struct {
+	Name        string `yaml:"name"`
+	Url         string `yaml:"url"`
+	Commit      string `yaml:"commit"`
+	Release_tag string `yaml:"release_tag"`
+}
+
+type ComponentsConfig struct {
+	Components []Component
+}
+
+func ParseComponentsYaml(componentsYaml string) ([]Component, error) {
```

Comment:
componentsConfigPath

It would make it clearer that it is a path to the file. Using Config instead of Yaml would put aside the format of the file which is not too relevant for the caller.